src/config/config.ts

```diff
@@ -1,76 +1,70 @@
 import { parseLocations } from "../utils/itemUtils"
+import type { AppConfig, Environment } from "../types/appTypes"
 
-export const appConfig = {
-  environment: process.env.APP_ENV || "production",
-  apiUrls: {
+export const appConfig: AppConfig = {
+  environment: (process.env.APP_ENV as Environment) || "development",
+  apiEndpoints: {
     platform: {
-      development:
-        process.env.PLATFORM_API_BASE_URL ||
-        "https://qa-platform.nypl.org/api/v0.1",
-      production:
-        process.env.PLATFORM_API_BASE_URL ||
-        "https://platform.nypl.org/api/v0.1",
+      development: "https://platform.nypl.org/api/v0.1",
+      qa: "https://qa-platform.nypl.org/api/v0.1",
+      production: "https://platform.nypl.org/api/v0.1",
     },
     // The 'discovery' base URL should use DISCOVERY_API_BASE_URL if set,
     // falling back on PLATFORM_API_BASE_URL if set,
     // and finally falling back on a sensible default.
     discovery: {
-      development:
-        process.env.DISCOVERY_API_BASE_URL ||
-        process.env.PLATFORM_API_BASE_URL ||
-        "https://qa-platform.nypl.org/api/v0.1",
-      production:
-        process.env.DISCOVERY_API_BASE_URL ||
-        process.env.PLATFORM_API_BASE_URL ||
-        "https://platform.nypl.org/api/v0.1",
+      development: "https://platform.nypl.org/api/v0.1",
```

Comment:
i think the platform url needs to match the auth url. otherwise we're going to have inconsistencies with running hold request and checkout flows. there are a bunch of records that are not the same or not present in both qa-platform and prod platform (likewise sierra). i'm having trouble putting my finger exactly on what would happen, but i don't think its a great idea. hold request renewals and cancellations will be happening in sierra test, but hold requests from RC, which happen via platform ASAIK, will be in prod. the patron will be populated with data from test, and i can see a bunch of weirdness going down.

tl;dr: beware mixing prod data with test auth
@nonword can you confirm?

Reply:
Agreed. I think you prob don't want to wire up local `development` environment to use `production` endpoints anywhere. The `discovery` base URL will control where holds are placed. And will likely also be where patron records are pulled from. When patron-login is folded in, the login/logout endpoints will need to agree with where we pull patron records from. So is there a reason to wire up local development to use production endpoints for any of these resources?

Reply:
I configured it that way to keep the settings consistent with the local config defaults before i made these changes. Makes sense the have development point to the QA endpoints, will make that change now.